<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="Logs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Servidor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de usuario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputServidor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNombre">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonConectar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonDesconectar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Desconectar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputMensaje">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEnviar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPuerto">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuServidores">
     <property name="title">
      <string>Servidores</string>
     </property>
     <addaction name="actionLista"/>
     <addaction name="actionA_adir"/>
    </widget>
    <addaction name="actionLista_de_usuarios"/>
    <addaction name="menuServidores"/>
    <addaction name="separator"/>
    <addaction name="actionLimpiar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLista_de_usuarios">
   <property name="text">
    <string>Lista de Usuarios</string>
   </property>
  </action>
  <action name="actionLimpiar">
   <property name="text">
    <string>Limpiar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionLista">
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionA_adir">
   <property name="text">
    <string>AÃ±adir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
